scripts/bake-scalable-azure-image-pipeline.ps1

```diff
@@ -56,7 +56,7 @@ param (
 
     [Parameter(Mandatory=$false)]
     [string]
-    $GitUserName="robe070",
+    $GitUserName="feature/aws-Build-Cloud-Account-Id-Artefacts-3.0",
```

Comment:
NO. Back out this change. And pass in the GitUserName from the YAML.

Reply:
done